bookkeeper-http/http-server/src/main/java/org/apache/bookkeeper/http/HttpRouter.java

```diff
@@ -31,11 +31,11 @@
 
   // Define endpoints here.
   public static final String HEARTBEAT                    = "/heartbeat";
-  public static final String SERVER_CONFIG                = "/api/config/serverConfig";
+  public static final String SERVER_CONFIG                = "/api/v1/config/server_config";
```

Comment:
Curious why hardcoded version here???

Reply:
I am not sure if I understand your question correctly. I think in restful world, there two common practices to deal with api versioning, one is putting version number in the url (so it is more explicitly and you know what you are calling), the other one is using accept-header, which version number is hidden in the accept header, it might make url looks better.

I suggest Jia does this way for making things explicit. I guess either way is fine, because in http, it is easy to add redirection.

Reply:
Thanks for the explain @sijie . 
@jvrao , Do you mean, we should make heartbeat the same format?

Reply:
Thanks @eolivelli